### DISCUSSION TITLE
Change split window size?

### DISCUSSION THREAD

**Discussion by marc-hg** [dalyIsaac/Whim#865] [Q&A] [answered]
I have two windows, side by side

I click, hold and drag in the middle to make the right one smaller horizontally, I can see it moving but when I let go it goes back to it's fixed place (50-50)

What am I missing here?

**Comment by urob** [comment, marked as answer]
This depends on the layout engine that you are using. For the `tree` layout, resizing should work (both via mouse and via keybinds). 

For the `slice` layout engine, resizing is planned (see #738), but currently not yet implemented. As a workaround, you could add a few different `sliceLayouts` to your config with different master column widths, and then toggle between them by switching layouts.

**Reply by marc-hg** [reply]
Thanks tree is what I was looking for
